lib/execution_engine2/utils/application_info.py

```diff
@@ -2,15 +2,19 @@
 Contains information about KBase applications.
 """
 
+from typing import Union
 from execution_engine2.utils.arg_processing import check_string as _check_string
 from execution_engine2.exceptions import IncorrectParamsException
 
 
-def _get2part_string(s, sep, name):
+def _get2part_string(s, sep, name, err_pt1, err_pt2, desired_sep=None):
```

Comment:
`desired_sep` is so the error message encourages the user to use a `/` separator rather then reinforcing the use of the incorrect `.` separator.